### PR TITLE
NXP-28808: add retention pages translation (Backport 10.10)

### PR DESCRIPTION
**Pull request by gcarlin**

### CODE REVIEW

**Comment by gcarlin** [comment]
Commit message updated

File: i18n/messages.json [hunk 1]
```diff
@@ -111,6 +111,9 @@
   "app.title.favorites": "Favorites",
   "app.title.home": "Home",
   "app.title.profile": "Profile",
+  "app.title.retentionEvents" : "Retention Events",
+  "app.title.retentionRules" : "Retention Rules",
+  "app.title.retentionSearch" : "Retention Search",
```
Comment (RSalem07) [inline i18n/messages.json:116]: no space between `"` and `:`:
`"app.title.retentionEvents" : "Retention Events",` should be `"app.title.retentionEvents": "Retention Events",`

Comment (RSalem07) [reply i18n/messages.json:116]: same thing for the different modifications ;)

Comment (gcarlin) [reply i18n/messages.json]: Fixed

**Pull request merged**